docs/admin/continuous.rst

```diff
@@ -64,7 +64,7 @@ source.
 
 * Enable :setting:`AUTO_UPDATE` to automatically update all components on your Weblate instance
 
-* Execute :djadmin:`updategit` (with selection of project or `--all` to update all)
+* Execute :djadmin:`updategit` (with selection of project or `\--all` to update all)
```

Comment:
Maybe it would be better this way? Not sure whether that renders correctly, though:

```suggestion
* Execute :djadmin:`updategit` (with selection of project or ``--all`` to update all)
```

Reply:
![Captura de tela de 2021-04-23 16-50-33](https://user-images.githubusercontent.com/1571783/115923068-14e9b200-a454-11eb-81ec-f0f3ce062081.png)

No problem here. I'll update it later (no git right now)

Reply:
Force-pushed a commit with your suggestion